libcontainer/integration/exec_test.go

```diff
@@ -1458,8 +1456,8 @@ func TestInitJoinPID(t *testing.T) {
 		Init:  true,
 	}
 	err = container2.Run(init2)
-	_ = stdinR2.Close()
```

Comment:
Honestly a bit on the fence on these; some linters may not complain about them, but (e.g.) my IDE may still. In the end we are discarding an error, and `_ =` makes it explicit that we know that's the case, and considered that it's ok to ignore.

Reply:
Sure, removed.